extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/index.tsx

```diff
@@ -336,12 +336,7 @@ export const SubmitFail = () => {
           "Destination account does not accept this asset",
         );
         errorDetails.errorBlock = (
-          <Notification
-            variant="error"
-            title={t(
-              "The destination account does not accept the asset you’re sending",
-            )}
-          >
+          <Notification variant="error" title="">
```

Comment:
I happened to hit this while testing and notice that we display the same text as the title several times on the error page already. It seemed appropriate enough to leave the title off from the notification since the layout around it includes a title already.
Before:
<img width="376" alt="Screenshot 2024-03-15 at 10 55 41 AM" src="https://github.com/stellar/freighter/assets/6886006/ef7c4933-9f19-4a6b-aa6f-c57f223adede">

After:
<img width="382" alt="Screenshot 2024-03-15 at 10 57 50 AM" src="https://github.com/stellar/freighter/assets/6886006/02907874-30c8-4552-9951-dc35b04dfba1">

Let me know if you think this should be different though.

Reply:
Oh yeah, we had a similar issue with another notification: https://github.com/stellar/freighter/blob/master/extension/src/popup/components/sendPayment/SendTo/index.tsx#L63

We actually handled by leaving the title and just omitting the repeated line in the body of the text

Reply:
Also, the `key ... returned an object...` error is a weird issue with `react-i18next` where if you try to translate a single line that exists verbatim in another body of text, it shows that error. So it's best in these cases to make sure the title text doesn't repeat in the text body

Reply:
thanks, makes sense. Updated in f8c6d5b4a261db0e6b04dabd31da94e399928932